modules/Bio/EnsEMBL/Compara/PipeConfig/ProteinTrees_conf.pm

```diff
@@ -628,7 +626,7 @@ sub core_pipeline_analyses {
             -module     => 'Bio::EnsEMBL::Compara::RunnableDB::AssertMatchingVersions',
             -input_ids  => [ { } ],
             -flow_into  => {
-                '1->A'  => [ 'copy_ncbi_tables_factory' ],
+                '1->A'  => [ 'load_mlss_id' ],
```

Comment:
At the moment, the pipeline doesn't require a master database and has a switch after `check_member_db_is_same_version`. `LoadMLSSids` does require a master database, so by inserting it so early you're removing the ability to run the pipeline without a master database. You should insert it further down instead, for instance just before `load_genomedb_factory`.